frontend/src/metabase/common/components/FilterPicker/ColumnValuesWidget/ColumnValuesWidget.tsx

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from "react";
+import FieldValuesWidget from "metabase/components/FieldValuesWidget";
+import * as Lib from "metabase-lib";
+import type Metadata from "metabase-lib/metadata/Metadata";
+
+interface ColumnValuesWidgetProps<T> {
+  value: T[];
+  column: Lib.ColumnMetadata;
+  metadata: Metadata;
+  canHaveManyValues?: boolean;
+  onChange: (value: T[]) => void;
+}
+
+export function ColumnValuesWidget<T extends string | number>({
+  value,
+  column,
+  metadata,
+  canHaveManyValues,
+  onChange,
+}: ColumnValuesWidgetProps<T>) {
```

Comment:
WDYT about grabbing `metadata` from Redux with `useSelector` instead of passing it down in each picker component?

Reply:
Sure, I can do that

Reply:
@kulyk done